docs/src/content/hardhat-runner/docs/supporter-guides/oracles.md

```diff
@@ -27,7 +27,7 @@ We can avoid the oracle problem by using a decentralized oracle network, which p
 
 ## Security
 
-An oracle is only as secure as its data source(s). If a dapp uses Uniswap as an oracle for its ETH/DAI price feed, an attacker can move the price on Uniswap to manipulate the dapp's understanding of the current price. An example of how to combat this is [a feed system](https://developer.makerdao.com/feeds/) like the one used by MakerDAO, which collates price data from many external price feeds instead of just relying on a single source.
+An oracle is only as secure as its data source(s). If a dapp uses Uniswap as an oracle for its ETH/DAI price feed, an attacker can move the price on Uniswap to manipulate the dapp's understanding of the current price. An example of how to combat this is [a feed system](https://docs.makerdao.com/smart-contract-modules/oracle-module/oracle-security-module-osm-detailed-documentation) like the one used by MakerDAO, which collates price data from many external price feeds instead of just relying on a single source.
```

Comment:
```suggestion
An oracle is only as secure as its data source(s). If a dapp uses Uniswap as an oracle for its ETH/DAI price feed, an attacker can move the price on Uniswap to manipulate the dapp's understanding of the current price. An example of how to combat this is [a feed system](https://docs.makerdao.com/smart-contract-modules/oracle-module/median-detailed-documentation) like the one used by MakerDAO, which collates price data from many external price feeds instead of just relying on a single source.
```

Good catch. I think the Median module description from the Maker docs is more appropriate as a link though.